scripts/install-jq.js

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env node
+
+'use strict'
+
+var which = require('which')
+var wget = require('download')
+var fs = require('fs')
+var path = require('path')
+
+var JQ_NAME = 'jq'
+
+var JQ_RELEASE_INFO = {
+  url: 'http://github.com/stedolan/jq/releases/download/',
+  version: 'jq-1.5'
+}
+
+var DOWNLOAD_URL = JQ_RELEASE_INFO.url + JQ_RELEASE_INFO.version + '/jq-'
+var ROOT_PATH = path.join(__dirname, '..')
+var BINARY_PATH = path.join(ROOT_PATH, 'node_modules/.bin/')
+var ORIGINAL_PATH = process.env.PATH
+
+/**
+ * Returns a clean path that helps avoid `which` finding bin files installed
+ * by NPM for this repo.
+ * @param {string} path
+ * @return {string}
+ */
+
+function cleanPath (path) {
+  return path
+      .replace(/:[^:]*node_modules[^:]*/g, '')
+      .replace(/(^|:)\.\/bin(:|$)/g, ':')
+      .replace(/^:+/, '')
+      .replace(/:+$/, '')
+}
+
+process.env.PATH = cleanPath(ORIGINAL_PATH)
+
+function findWhereIsInstalled (binary) {
+  return which.sync(binary)
+}
+
+function findGlobaly (binary) {
+  return !!findWhereIsInstalled(binary)
```

Comment:
Oh no, casting to boolean 😭 

You can do a try/catch here for `getNotFoundError` (thrown [here](https://github.com/npm/node-which/blob/master/which.js#L132))